easybuild/easyblocks/t/tensorflow.py

```diff
@@ -374,16 +379,17 @@ def build_step(self):
         if cuda_root:
             cmd.append('--config=cuda')
 
-        # if mkl-dnn is listed as a dependency it is used. Otherwise downloaded if with_mkl_dnn is true
-        mkl_root = get_software_root('mkl-dnn')
-        if mkl_root:
-            cmd.extend(['--config=mkl'])
-            cmd.insert(0, "export TF_MKL_DOWNLOAD=0 &&")
-            cmd.insert(0, "export TF_MKL_ROOT=%s &&" % mkl_root)
-        elif self.cfg['with_mkl_dnn']:
-            # this makes TensorFlow use mkl-dnn (cfr. https://github.com/01org/mkl-dnn)
-            cmd.extend(['--config=mkl'])
-            cmd.insert(0, "export TF_MKL_DOWNLOAD=1 && ")
+        if get_cpu_architecture() == X86_64:
```

Comment:
I'm not sure this is actually the right way to go here...

Do we *always* want to skip on `mkl-dnn` entirely on non-x86_64 systems (also in the future)? Note that this is broader than "not on POWER"...

I think this change should be reverted to what it was, and only change the default value for `with_mkl_dnn`.

On POWER systems, you can then avoid the use of `mkl-dnn` by configuring EasyBuild with `--filter-deps=mkl-dnn`?

Unless you're confident it will never make sense to even consider `mkl-dnn` on non-x86_64 systems, then this is ok :)

Reply:
That's how I had it to start with :-) I'll put it back.